implementation/leprechaun/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "declaration": true,
+    "removeComments": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "ES2021",
+    "sourceMap": true,
+    "outDir": "./dist",
+    "baseUrl": "./",
+    "incremental": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "strictNullChecks": false,
+    "noImplicitAny": false,
+    "strictBindCallApply": false,
+    "forceConsistentCasingInFileNames": false,
+    "noFallthroughCasesInSwitch": false
+  }
```

Comment:
Wait, so you enabled `strict`, which is a combination of multiple typechecking options and then manually disabled almost all of the included options, making `strict` useless? 😠

Reply:
I've enabled `strict`, removed your `: false` checks and ran type checking. And I can confirm that all of the errors you are getting are not false flags, and are caused by possibly unsafe operations.

Reply:
`nestjs` asks when creating project, whether to enable strict checking, and those values came from that. I didn't know tbh that strict is just alias for enabling some options. Seems kinda uneccessary for `nestjs` to set them explicitly instead of setting `strict` to false.